terraform/gcp/projects/awi-ciroh.tfvars

```diff
@@ -68,6 +68,23 @@ notebook_nodes = {
       count: 0
     }
   },
+  # added stressfully before an event where we ran out of ssd quota, see
+  # https://github.com/2i2c-org/infrastructure/pull/2539.
```

Comment:
Ditto link to event issue not PR.

Reply:
I want to link to the PR where I posted screenshots about the failures related to ssd quota though. I added a commit to link to the event as well directly.